kernel/src/tbfheader.rs

```diff
@@ -531,6 +612,15 @@ crate fn parse_tbf_header(header: &'static [u8], version: u16) -> Result<TbfHead
                                 .or(Err(TbfParseError::BadProcessName))?;
                         }
 
+                        TbfHeaderTypes::TbfHeaderFixedAddresses => {
+                            let entry_len = mem::size_of::<TbfHeaderV2FixedAddresses>();
```

Comment:
TbfHeaderV2FixedAddresses needs to be `repr(C)` for this check to make sense. It probably makes more sense to hard-code 8 here.